core/src/main/java/io/kestra/core/models/triggers/types/Schedule.java

```diff
@@ -445,7 +457,7 @@ private boolean validateScheduleCondition(ConditionContext conditionContext) {
     @EqualsAndHashCode
     @Getter
     @NoArgsConstructor
-    public static class Output implements io.kestra.core.models.tasks.Output {
+    public static class ScheduleDates implements io.kestra.core.models.tasks.Output {
```

Comment:
Why do you change the name of the output? It is more or less a standard to use Output for the name of the output.

Reply:
I thought it was more explicite to have "scheduledDates" (it even missed a d here) than Output but I can set it back
But I just realized it will probably have impact on the variable name

Reply:
It has no impact on the var names but it makes a lot of changes for not much improvements as we often name the Ouput of a task Output ...